app/models/miq_retire_task.rb

```diff
@@ -87,4 +87,17 @@ def completed_state
   def self.display_name(number = 1)
     n_('Retire Task', 'Retire Tasks', number)
   end
+
+  def workflow_inputs
+    source_details = {
+      :name    => source.name,
```

Comment:
Some other fields we may want to consider
- id
- href (UI and API - maybe api_href and ui_href?)
- service id/href?

The hrefs I could see being useful to construct links in emails or tickets.